nle/scripts/play.py

```diff
@@ -100,10 +101,14 @@ def play(env, mode, ngames, max_steps, seeds, savedir, no_render, render_mode, d
     while True:
         if not no_render:
             if not is_raw_env:
-                print("Previous reward:", reward)
-                if action is not None:
-                    print("Previous action: %s" % repr(env._actions[action]))
-                env.render(render_mode)
+                # print('--------')
+                # print(f"Previous reward: {str(reward):64s}")
+                act_str = repr(env._actions[action]) if action is not None else ''
+                # print(f"Previous action: {str(act_str):64s}" )
+                # print('--------')
+                # env.render(render_mode)
+                # print('--------')
+                # print('\033[31A') # Go up 31 lines.
```

Comment:
Are these lines supposed to be commented out? (also the ones below)

Reply:
will adjust these

Reply:
done